.github/workflows/macos_test.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: 🍎 MacOS Tests
+
+on:
+  workflow_dispatch:
+
+jobs:
+  tests:
+    env:
+      COLORTERM: truecolor
+      NPROC: 2
+      TERM: xterm
+    runs-on: macos-latest
+
+    steps:
+
+      - name: Install tools and libraries via Homebrew
+        if: matrix.platform.os == 'macos'
```

Comment:
No matrix is defined in this workflow so the `if:` would bomb out, I think. But since each workflow in your PR is per platform, it could/should be removed anyway.

Reply:
Should be fixed now.